Code/GraphMol/FMCS/FMCS.cpp

```diff
@@ -236,6 +238,24 @@ bool checkAtomChirality(const MCSAtomCompareParameters& p,
   return true;
 }
 
+bool checkAtomDistance(const MCSAtomCompareParameters& p,
+                       const ROMol& mol1, unsigned int atom1,
+                       const ROMol& mol2, unsigned int atom2){
+  unsigned int confId1 = 0;
+  unsigned int confId2 = 0;
+  if (mol1.getNumConformers() <= confId1 ||
+      mol2.getNumConformers() <= confId2){
+    throw std::runtime_error(
+        "FMCS. Invalid argument. requested conformer unavailable");
+  }
```

Comment:
This isn't required. When if you call `getConformer()` with either a bad conformer ID or when no conformers are present, it will thrown the appropriate exception

Reply:
Changed